docs/development/testing-styleguide.md

```diff
@@ -11,6 +11,15 @@ We use ~~"Futurelearn style" acceptance tests~~ This blog post has been removed
 3. Define all steps in the file. If you want to share code between scenarios, call helpers that are defined in a module from the step.
 4. The steps should be written in English. Do not use parameters to call the step methods.
 
+## Timetravel
+
+In CI, we run our test suite with offset-dates to ensure tests won't break as we encounter different milestones in the cycle.
+If you are writing a test for something that only works during some points in the cycle, you will need to make that explicit in your test by setting the time in the describe / scenario block. eg `time: mid_cycle`
```

Comment:
How do you feel about adding a link to an example of one our tests that has this setup?

Reply:
I've added it to the example in this document -- I am hesitant to add a link to a specific example here as it will need to be maintained. I would expect a developer to just search the codebase for `time: mid_cycle` and find some examples for themselves.